light-system-programs/tests/merkle_tree_tests.ts

```diff
@@ -91,11 +92,11 @@ describe("Merkle Tree Tests", () => {
       2_000_000_000,
     );
 
-    RELAYER = new TestRelayer({
+    RELAYER = await new TestRelayer({
```

Comment:
why an await here?

Reply:
god find, my bad, copy-pasted the relayer setup